pkg/controller/awssecret/awssecret_controller.go

```diff
@@ -53,15 +49,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// TODO(user): Modify this to be the types you create that are owned by the primary resource
-	// Watch for changes to secondary resource Pods and requeue the owner AWSSecret
-	err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &mumoshuv1alpha1.AWSSecret{},
-	})
-	if err != nil {
-		return err
-	}
+// i'm not entirely sure what this section is for, as we don't create any pods.
```

Comment:
Good catch, seems like I've left this as-is unintentionally after generating the skaffold via operator-sdk.